package.json

```diff
@@ -30,10 +30,10 @@
     "stylelint-selector-no-utility": "^1.3.0"
   },
   "devDependencies": {
-    "ava": "^0.17.0",
-    "eslint": "^3.0.1",
-    "eslint-plugin-github": "^0.7.0",
-    "stylelint": "^7.4.2"
+    "ava": "*",
+    "eslint": "*",
+    "eslint-plugin-github": "*",
+    "stylelint": "*"
```

Comment:
This is just a move to remove numbers here, so that greenkeeper doesn't bother sending PRs for them.